python/voici/README.md

```diff
@@ -17,7 +17,26 @@ https://user-images.githubusercontent.com/591645/222892327-2a5b1341-640d-49c2-9e
 - Generates self-contained HTML files with embedded Wasm kernels.
 - Works offline, without requiring a server to run the dashboard.
 - Supports custom templates for styling dashboards, powered by Jinja2.
-- Supports all programming languages that have JupyterLite kernels available. _e.g._ the default JavaScript and Python kernels JupyterLite provides, [python with xeus-python](https://github.com/jupyterlite/xeus-python-kernel), [lua](https://github.com/jupyterlite/xeus-lua-kernel), [nelson](https://github.com/jupyterlite/xeus-nelson-kernel)
+- Supports all programming languages that have JupyterLite kernels available. _e.g._ the default JavaScript and Python kernels JupyterLite provides, and [xeus kernels](https://github.com/jupyterlite/xeus)
+
+## Difference between `voici` and `voici-core` packages
+
+- The `voici-core` package is the core functionalities of voici, mainly the voici CLI.
+- The `voici` package is a meta-package installing both `voici-core` and [`jupyterlite-xeus`](https://github.com/jupyterlite/xeus).
+
+`jupyterlite-xeus` allows you to pre-install packages for running your dashboard. For example, if your dashboard requires Matplotlib you can provide an `environment.yml` file in the folder where you run the voici command, containing the following:
```

Comment:
Maybe there could be a link to the `jupyterlite-xeus` documentation, and a quick note to explain why `voici` chooses to depend on `jupyterlite-xeus` by default?

Reply:
Also as discussed in https://github.com/voila-dashboards/voici/pull/106#issuecomment-1903558394, maybe this change can be documented in the changelog (or in a new migration guide)? For those who already have a Voici deployment.

Although normally this will probably be transparent to the users (especially those using Pyodide), because `jupyterlite-xeus` would not do much if there is no xeus kernel installed?

Reply:
> Maybe there could be a link to the `jupyterlite-xeus` documentation, and a quick note to explain why `voici` chooses to depend on `jupyterlite-xeus` by default?

Something saying that it helps supports other kernels such as `xeus-javascript`, `xeus-lua`, `xeus-nelson` and so on would also be nice.

Reply:
Thanks for the comment, I pushed another commit resolving them.

> Although normally this will probably be transparent to the users (especially those using Pyodide), because jupyterlite-xeus would not do much if there is no xeus kernel installed?

It should not impact them indeed. Let's add changelog notes during the release.